flake.nix

```diff
@@ -59,57 +59,13 @@
     // flake-utils.lib.eachDefaultSystem (
       system: let
         lib = pkgs.callPackage ./lib/lib.nix {
-          inherit system;
```

Comment:
This argument is still present and in use, please keep it around.

Reply:
`callPackage` seems to fill in the `system` parameter automatically
```bash
$ nix repl nixpkgs
nix-repl> legacyPackages.x86_64-linux.callPackage ({system}: system) {}
"x86_64-linux"
```
on my system. So if I'm not missing anything w.r.t. `system` being something else when its coming from `flake-utils`, this shouldn't be an issue

Reply:
Yes, you are correct. It basically ends up coming from line 65 here. No need to repeat it then! :)

Reply:
But since were on the topic of necessary arguments. is there a reason we can't just use `../.` inside `lib/lib.nix` instead of passing in `self.outPath`?